aten/src/ATen/native/SummaryOps.cpp

```diff
@@ -52,10 +51,78 @@ Tensor _bincount_cpu_template(
   }
   return output;
 }
+
+///////////////// histogram /////////////////
+template <typename input_t>
```

Comment:
Add a brief comment describing this function

Reply:
Done